doc/content/help/faq.md

```diff
@@ -1,14 +1,140 @@
+
+  - `FIRSTNAME.LASTNAME@inl.gov` is your INL email ID.
+  - `Example03_hpc` is the name of the job.
+  - `select=4:ncpus=36:mpiprocs=36` means you requested 4 nodes, 36 cpus, and 36 processors. You can increase or decrease these numbers depending on the hpc machine.
+  - `walltime=00:90:00` is the wall time above which the jobs gets terminated. The format is Days:Hours:Minutes. Larger the wall time, more the delay in getting the job started, in general.
+  - `MV2_ENABLE_AFFINITY=0 mpiexec ~/projects/mastodon/mastodon-opt -i shear_beam_Isoil_free_field.i >> out` has the name of the input file. Use the input file name you are interested in.
+
+- Finally, save this file `Example03_hpc.sh` in the folder in which the input file is located. Batch files can be submitted into the queue using the qsub command as shown below.
+
+```
+qsub Example03_hpc.sh
+```
+
+- For more information on PBS see [this](http://hpcweb.inl.gov/home/pbs).
+
+#### 4. Some or all my tests are failing, what should I do?
+
+There might be one of several reasons for this:
+
+- You did not compile MASTODON. Compile MASTODON using `make` command.
+
+
+- If the errors while running tests are related to libmesh, your libmesh is out of date. Update and rebuild libmesh using the following command in the MASTODON folder. Then compile MASTODON and try running the tests.
+
+```
+./moose/scripts/update_and_rebuild_libmesh.sh
+```
+
+- You might have accidentally made some changes to MASTODON source code. To undo these changes use the below command. Then update and rebuild libmesh, compile MASTODON, and run the tests as described in [Getting Started/MacOS (Mojave)](https://mooseframework.inl.gov/mastodon/getting_started/macos_mojave.html).  
+
+```
+git clean -dxf
+```
+
+Beware! Running this command will completely erase all additional files and uncommitted changes to the MASTODON directory.
+
+#### 5. How to write an input file from scratch?
+
+Generally, we do not recommend doing this. On the [Examples](https://www.mooseframework.org/mastodon/examples/index.html) page, a variety of problems related to seismic soil structure interaction of nuclear structures are discussed. We recommend downloading the input file from one of those problems and modifying it to suit your own problem. In this process, the MASTODON [User Manual](https://www.mooseframework.org/mastodon/manuals/user/index.html) and [Syntax](https://www.mooseframework.org/mastodon/syntax/index.html) pages might also help. There are also a number of input files checked into the `tests` directories within MASTODON.
+
+#### 6. How to input a mesh file from ABAQUS (i.e., a `.inp` file)?
+
+MASTODON is capable of accepting mesh files in many formats, one of them being an ABAQUS mesh file. Below is an example on how to input an ABAQUS mesh file. Please refer to [this](https://www.mooseframework.org/source/meshgenerators/FileMeshGenerator.html) MOOSE page for more information.
+
```

Comment:
Make a note here, that the .inp reader is limited to only one mesh block. If the .inp file contains multiple parts, they will all be moved into one block in MOOSE/MASTODON. This limitation will be fixed soon.